### ISSUE TITLE
explorer.zclassic.org

### ISSUE THREAD

**Issue by ch4ot1c** [z-classic/z-classic.github.io#50]
Gotta have it!

**Comment by nimbosa** [comment]
Yes, we have an insight-based explorer that should be up soon! We're working on this to bring it up again ASAP. The iquidus-based explorer was just a stop-gap measure, and both explorers will be migrated to a better infrastructure now that we're in this for the long haul.

But we need all the help we can get.  Thanks @ch4ot1c.

**Comment by mexican1973** [comment]
New explorers online now:
http://insight.zcl.community/
http://explorer.zcl.zeltrez.io/
